### PR TITLE
IOUTIL package deprecated

### LINKED ISSUE #757: ioutil readall function is deprecated after go 1.16

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

- IOUTIL package has been deprecated after go 1.16
- This has been removed in this PR and affected functions has been changed to the latest supporting packages.

Fixes #757

### CODE REVIEW

File: utils/struct-utils.go [hunk 1]
```diff
@@ -394,21 +403,6 @@ func (a ABIStruct) Pack(parsedData abi.ABI, name string, args ...interface{}) ([
...
-	return ioutil.ReadAll(body)
-}
-
-//This function is used to read the file
-func (i IoutilStruct) ReadFile(filename string) ([]byte, error) {
-	return ioutil.ReadFile(filename)
-}
-
-//This function is used to write data to file
-func (i IoutilStruct) WriteFile(filename string, data []byte, perm fs.FileMode) error {
```
Comment (ashish10677) [inline utils/struct-utils.go:408]: You've removed WriteFile, is it not added?

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: utils/struct-utils.go (hunk 1 above)
Comment (shekhar2807) [reply utils/struct-utils.go:408]: It's already there is struct utils. @ashish10677 
https://github.com/razor-network/razor-go/blob/41d326ad9374af7042931ddedf671ad20fbb1039/utils/struct-utils.go#L338

Earlier we had writeFile from both the packages inutile and os .

Comment (ashish10677) [reply utils/struct-utils.go:408]: Ok cool.

**Review by ashish10677 — APPROVED** [review]
LGTM

**Pull request merged**